arduino-ide-extension/src/browser/boards/boards-service-provider.ts

```diff
@@ -209,7 +220,7 @@ export class BoardsServiceProvider implements FrontendApplicationContribution {
     }
   }
 
-  protected async tryReconnect(): Promise<boolean> {
+  protected tryReconnect(): boolean {
```

Comment:
couldn't see a reason for this to be async, it's not called across the wire either